.github/workflows/ci.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         if: ${{ matrix.os == 'ubuntu-latest' }}
         run: echo "installing pinned version of setuptools-scm to fix seqeval installation on 3.7" && pip install "setuptools-scm==6.4.2"
       - name: Install uv
-        run: pip install --upgrade uv
+        run: pip install uv==0.1.29
```

Comment:
Would remove the pin to be consistent with `huggingface_hub` and `diffusers`:
```suggestion
```

(we don't use `uv`'s advanced/experimental features, so a breaking change here is unlikely)

Reply:
I had pinned it because 0.1.30 had bugs - I'll see if 0.1.31 has fixed them

Reply:
It's been fixed in https://github.com/astral-sh/uv/issues/2941 :)